snap/naming/tag.go

```diff
@@ -36,8 +36,8 @@ type ParsedSecurityTag interface {
 	InstanceName() string
 }
 
-// ParsedAppSecurityTag describes a parsed snap application security tag.
```

Comment:
should ParsedSecurityTag, also be just called SecurityTag ?

Reply:
I was wondering about this myself. It depends on if we want to have `type SecurityTag string` as well, and use it for extra type safety in cases where we do not want to parse the tag but actually generate it and pass it around.

Reply:
I wouldn't do that,  see SnapRef, it's probably easier to have a function that makes a parsed one out of the components and not the parsing, instead of having both `type SecurityTag string` which is a weak guarantee (you can always cast to it) and a ParsedSecurityTag interface

Reply:
In that case let’s rename the interface to SecurityTag.